src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -720,17 +720,16 @@ public virtual void Clear()
         ///     If successful, the CurrentWorkspace.FilePath field is updated as a side effect
         /// </summary>
         /// <param name="newPath">The path to save to</param>
-        /// <param name="core"></param>
         /// <param name="isBackup">Indicates whether saved workspace is backup or not. If it's not backup,
         /// we should add it to recent files. Otherwise leave it.</param>
-        public virtual bool SaveAs(string newPath, ProtoCore.RuntimeCore runtimeCore, bool isBackup = false)
```

Comment:
This method required a `RuntimeCore` due to trace erroneously living on `HomeWorkspaceModel`. I generalized this method (refactoring `SerializeSessionData` to `SaveCustomData`) and moved the implementation having to do with trace to where it belongs - `HomeWorkspaceModel`. Note that `DynamoModel` does not have a `RuntimeCore`, it lives on the `HomeWorkspaceModel`.